setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+setup(
+    name='mimic',
+    version="0.0.1",
+    packages=find_packages(),
+    install_requires=['docutils'],
```

Comment:
Then - since we don't care about version that much - yes, I'd propose that you remove requirements.txt then I'll merge.